### ISSUE TITLE
pmacros_test fails

### ISSUE THREAD

**Issue by dabaci** [saprykin/plibsys#94]
After building runnign `ctest` in `plibsys-build`, the pmacros_test failed. Running `ctest --rerun-failed --output-on-failure` outputs

```
Test project /path/to/plibsys-build
    Start 11: pmacros_test
1/1 Test #11: pmacros_test .....................***Failed    0.00 sec
Running test case: pmacros_general_test
/path/to/plibsys/tests/pmacros_test.cpp:326: check failed
** Warning: Test warning output **
** Error: Test error output **
** Debug: Test debug output **
** Debug: Likely condition triggered **
Test failed


0% tests passed, 1 tests failed out of 1

Total Test time (real) =   0.01 sec

The following tests FAILED:
	 11 - pmacros_test (Failed)
Errors while running CTest
```

I am on an M1 MacBook Pro.

**Comment by dabaci** [comment]
I took a look at `ptest_macros.c`, and I believe the error is happening because the test does not check for ARM architecture on computers running macOS (see line 325).

**Comment by saprykin** [comment]
This is indeed something new

**Comment by saprykin** [comment]
Should be fixed in 6301fbbdd7f141af139591193173b2609b7c918b.
